course05/homework01/skeleton_twyleg/string.h

```diff
@@ -0,0 +1,41 @@
+// Some copyright 2020
+
+#pragma once
+
+#include <string>
+#include <ostream>
```

Comment:
You only need std::ostream declaration in this file, not its definition - which means you don't need this entire / heavy include, you can use iosfwd, and include ostream from the cc only!

Reply:
- [ ] include ostream from the cc only!